pytac/units.py

```diff
@@ -4,29 +4,80 @@
 
 
 def unit_function(value):
+    """Default value for the pre and post functions used in unit conversion.
+
+    Args:
+        value(Number): The value to be converted.
+
+    Returns:
+        value(Number): The result of the conversion.
+    """
     return value
 
 
 class UnitConv(object):
     def __init__(self, post_eng_to_phys=unit_function, pre_phys_to_eng=unit_function):
+        """Class to convert between physics and engineering units.
+
+        The two arguments to this function represent functions that
+        are applied to the result of the initial conversion.
+
```

Comment:
One happens _after_ the conversion, the other happens _before_ the conversion back.